### DISCUSSION TITLE
Ability to sort a table server side where column is based on two values

### DISCUSSION THREAD

**Discussion by cdecinkoKnight** [quasarframework/quasar#16413] [General - Components / Directives / etc]
I have a city and state value coming back from my API. I display them in a single column much like Phoenix, AZ. The column is titled Origin. When I click on that column heading, I want to pass a value to my API that will return the results sorted first by state, then by city, i.e. orderBy=originState, originCity. I've tried several methods and every single time, the title Origin is what is passed to my API.

I tried a computed, I tried a custom sort and none seem to even fire. The sort of "originState, originCity" is never passed.

**Comment by metalsadman** [comment]
it sends what you use as the `name` prop of the column object. 
ps please add more details like some code snippet.
